css/styles.css

```diff
@@ -0,0 +1,52 @@
+body 	{
```

Comment:
@gerriecho Very nice site! Love how the image is the right half of the screen! Responsive, too. Nice.